### PR TITLE
Remove random requests from mergify and add as GH action.

### PR DESCRIPTION
**Pull request by destogl**

### CODE REVIEW

File: .github/reviewer-lottery.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - name: maintainers # name of the group
+    reviewers: 2 # how many reviewers do you want to assign?
+    internal_reviewers: 1 # how many reviewers do you want to assign when the PR author belongs to this group?
+    usernames: # github usernames of the reviewers
+      - bmagyar
+      - destogl
+
+  # Reviewers group to get broader feedback.
+  - name: reviewers
+    reviewers: 5
```
Comment (bmagyar) [inline .github/reviewer-lottery.yml:14]: 5 is a little steep, no? 2 should be enough

File: .github/reviewer-lottery.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      - peterdavidfagan
+      - UsamaHamayun1
+      - bijoua29
+      - Aposhian
+      - safdark
+      - rk800506
+      - chrismath
+      - lauesa
+      - gokulmaxi
+      - Briancbn
```
Comment (bmagyar) [inline .github/reviewer-lottery.yml:44]:
```suggestion
      - rosterloh
      - kellyprankin
      - progtologist
      - arne48
      - DasRoteSkelett
      - a10263790
      - Serafadam
      - harderthan
      - jaron-l
      - malapatiravi
      - erickisos
      - ShawnSchaerer
      - Briancbn
      - TomoyaFujita2016
      - homalozoa
      - anfemosa
      - jackcenter
      - VX792
      - mhubii
      - livanov93
      - aprotyas
      - peterdavidfagan
      - UsamaHamayun1
      - duringhof
      - bijoua29
      - kasiceo
```

Comment (bmagyar) [reply .github/reviewer-lottery.yml:44]: updated with the list of reviewers as per this morning (not adding the ones who were only invited but didn't accept)

**Review by bmagyar — CHANGES_REQUESTED** [review]

File: .github/reviewer-lottery.yml (hunk 1 above)
Comment (destogl) [reply .github/reviewer-lottery.yml:14]: we have some many people. Let's see how this works. They would anyway have a lot things to do. Don't you think?

Comment (bmagyar) [reply .github/reviewer-lottery.yml:14]: ok

**Review by bmagyar — APPROVED** [review]

**Pull request merged**